EKF/estimator_interface.h

```diff
@@ -91,6 +91,11 @@ class EstimatorInterface
 	virtual void getFlowInnov(float flow_innov[2]) const = 0;
 	virtual void getFlowInnovVar(float flow_innov_var[2]) const = 0;
 	virtual void getFlowInnovRatio(float &flow_innov_ratio) const = 0;
+	virtual Vector2f getFlowVelBody() const = 0;
+	virtual Vector2f getFlowVelNE() const = 0;
+	virtual Vector2f getFlowCompensated() const = 0;
```

Comment:
Are those all really needed?

Reply:
I don't think so; the getters could be directly in the estimator interface, but I preferred having them next to the `getFlowInnov...`. We could then move them to the estimator interface altogether in a separate PR.